README.md

```diff
@@ -4,12 +4,31 @@ Notation is a project to add signatures as standard items in the registry ecosys
 
 ## Table of Contents
 
+- [Notation Quick Start](#notation-quick-start)
 - [Branch](#branch)
 - [Core Documents](#core-documents)
 - [Community](#community)
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## Notation Quick Start
+
+- Install the Notation CLI from [Notation Releases][notation-releases]  
+  ```bash
+  curl -Lo notation.tar.gz https://github.com/shizhMSFT/notation/releases/download/v0.5.2/notation_0.5.2_linux_amd64.tar.gz
+  tar xvzf notation.tar.gz -C ~/bin notation
+  ```
+- Build, Push, Sign, Verify the `net-monitor` software
+  ```bash
+  export IMAGE=localhost:5000/net-monitor:v1
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+  docker push $IMAGE
+  notation cert generate-test --default --trust "wabbit-networks-dev"
+  notation sign $IMAGE
```

Comment:
Do we want to demonstrate push/pull signatures from repository?

Reply:
`notation sign` defaults to include push. There are separate apis for generating an offline signature, which we'd cover in a different walkthrough for offline/ephemeral client scenarios.